tools/storybook-addon/package.json

```diff
@@ -31,22 +31,23 @@
     "@lg-tools/storybook-decorators": "workspace:^",
     "@lg-tools/storybook-utils": "workspace:^",
     "@mdx-js/react": "2.3.0",
-    "@storybook/addon-a11y": "7.6.20",
-    "@storybook/addon-essentials": "7.6.20",
-    "@storybook/addon-interactions": "7.6.20",
-    "@storybook/addon-links": "7.6.20",
-    "@storybook/addon-mdx-gfm": "7.6.20",
-    "@storybook/addons": "7.6.20",
-    "@storybook/api": "7.6.20",
-    "@storybook/components": "7.6.20",
-    "@storybook/core-events": "7.6.20",
+    "@storybook/addon-a11y": "8.5.3",
+    "@storybook/addon-essentials": "8.5.3",
+    "@storybook/addon-interactions": "8.5.3",
+    "@storybook/addon-links": "8.5.3",
+    "@storybook/addon-mdx-gfm": "8.5.3",
+    "@storybook/components": "8.5.3",
+    "@storybook/core-events": "8.5.3",
     "@storybook/jest": "^0.2.3",
-    "@storybook/test": "^7.6.20",
-    "@storybook/react": "7.6.20",
-    "@storybook/react-webpack5": "7.6.20",
-    "@storybook/testing-library": "0.2.0",
-    "@storybook/theming": "7.6.20",
+    "@storybook/manager-api": "8.5.3",
+    "@storybook/preview-api": "8.5.3",
```

Comment:
`/api` and `/addons` were soft-deprecated in favor of these packages
https://storybook.js.org/docs/migration-guide#package-structure-changes